.circleci/config.yml

```diff
@@ -4,10 +4,28 @@ orbs:
   terraform: circleci/terraform@2.1.0
 
 executors:
-  linux-machine:
+  cross-builder:
+    docker:
+      - image: quay.io/influxdb/cross-builder:go1.17.1-41d944d749b2c130119eff7efaaa9883ccffe5a7
+    resource_class: large
+  linux-amd64:
     machine:
       image: ubuntu-2004:202107-02
       resource_class: large
+  linux-arm64:
+    machine:
+      image: ubuntu-2004:202101-01
+      resource_class: arm.large
+  mac:
```

Comment:
This is probably a bit too pendantic, but what do you think about using `darwin` instead of `mac`, since that's how we identify that target for our packages?